.travis.yml

```diff
@@ -22,6 +22,8 @@ branches:
 before_install:
   - pip install -U pip wheel setuptools
   - pip install --no-cache-dir -U -r requirements-test.txt
+  - if [[ $TRAVIS_PYTHON_VERSION == "3.6" ]]; then pip install "django>=2.0,<2.1"; fi
+  - if [[ $TRAVIS_PYTHON_VERSION == "2.7" ]]; then pip install "django>=1.11,<2.0"; fi
```

Comment:
move these two lines to the beginning of the `install` section of `.travis.yml` in order to not slow down the style checks which need to be performed very fast